grafana-plugin/src/models/loader/loader.ts

```diff
@@ -0,0 +1,20 @@
+import { observable } from 'mobx';
+
+interface LoadingResult {
+  [key: string]: boolean;
+}
+
+class LoaderStoreClass {
+  @observable
+  items: LoadingResult = {};
+
+  setLoadingAction(actionKey: string, isLoading: boolean) {
```

Comment:
This should be annotated as @action

Reply:
I think we should support LoadingState as union, not boolean. We might have more statuses, e.g. error. Example here: https://github.com/brojd/todo-app/blob/main/frontend/src/store/store.types.ts